services/wallet/activity/session.go

```diff
@@ -18,12 +17,28 @@ import (
 	"github.com/status-im/status-go/transactions"
 )
 
+const nilStr = "nil"
+
 type EntryIdentity struct {
 	payloadType PayloadType
 	transaction *transfer.TransactionIdentity
 	id          transfer.MultiTransactionIDType
 }
 
+// func (e EntryIdentity) same(a EntryIdentity) bool {
```

Comment:
can this be removed?

Reply:
I revived and use it in one of my folowing PRs :)